src/Context/LikedVideosContext.jsx

```diff
@@ -0,0 +1,68 @@
+import { createContext, useContext, useReducer, useState } from "react";
+import axios from "axios";
+import { useAlert } from "./AlertContext";
+import { v4 as uuid } from "uuid";
+import { actionAddToLikedVideosList } from "../Helpers/Services/actions";
+
+
+//CONTEXT CREATED
+const LikedVideosContext = createContext();
+
+const LikedVideosProvider = ({ children }) => {
+
+    const encodedToken = localStorage.getItem("authData");
+    const [likedVideos , setLikedVideos] = useState([]);
+    const {alertContent , setAlertContent} = useAlert();
+
+    const addToLikedVideos = async (video) => {
```

Comment:
Suggestion: I am not sure but i think this could be a utilty function

Reply:
utilty function for ??